### PR TITLE
feature/advice

### PR DESCRIPTION
**Pull request by Bulrati**
Makefile was added, composer was added, .env.example was added

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ### Docker-related stuff ###
 .env*
```
Comment (sigma-victor-bairak) [inline .gitignore]: Only files that might be changed should be ignored. Usually it's only `.env`.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+restart:
+	@make stop
+	@make start
+
+ssh:
+	@docker exec -it $(PHPFPM_CONTAINER) sh
+
+build:
+	@docker-compose up -d --build
```
Comment (sigma-victor-bairak) [inline Makefile]: It's recommend to put empty line at the end of files. Check all files that you've added
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

File: .env.example [hunk 3]
```diff
@@ -7,7 +7,7 @@ APP_URL=http://localhost
 LOG_CHANNEL=stack
 
 DB_CONNECTION=mysql
-DB_HOST=database
+DB_HOST=localhost
```
Comment (sigma-victor-bairak) [inline .env.example]: `database` is correct value, with `localhost` it won't work as there is no installed mysql server in phpfpm container where laravel app is being run.

File: database/migrations/2019_11_30_092123_create_post_statuses_table.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+use Illuminate\Support\Facades\Schema;
+
+class CreatePostStatusesTable extends Migration {
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up() {
+		Schema::create( 'post_statuses', function ( Blueprint $table ) {
```
Comment (sigma-victor-bairak) [inline database/migrations/2019_11_30_092123_create_post_statuses_table.php]: Please follow PSR standards during implementation of laravel applications. No extra spaces within parentheses `()` of methods, braces of methods should be on separate lines.
https://svyatoslav.biz/misc/psr_translation/#_PSR-1
Please configure inspection rules and auto-formatting of your IDE to meet PSR requirements

File: .env.example [hunk 5]
```diff
@@ -0,0 +1,44 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:BRIrrEIO+jBwBksEztbwyPoE6r7/wCQjTisadad9oXM=
+APP_DEBUG=true
+APP_URL=http://localhost
+
+LOG_CHANNEL=stack
+
+DB_CONNECTION=mysql
+DB_HOST=road-database
```
Comment (sigma-victor-bairak) [inline .env.example:10]: Host should be as a name of service (`database` from docker-compose), not a container. Please check whether it works before commit.

Comment (Bulrati) [reply .env.example:10]: It works even with container name, I've just forgotten to add commands to makefile to run migrations, but thank you now I know that I should use service name.

Fixed.

Comment (sigma-victor-bairak) [reply .env.example:10]: Hm... Didn't know about it. Thanks!

File: docker/phpfpm/DockerFile [hunk 6]
```diff
@@ -1,8 +1,11 @@
-FROM php:7.3-fpm
+FROM php:7.3-fpm-alpine
 
-RUN apt-get update && apt-get install -y wget git unzip \
+RUN apk add --no-cache \
+		g++ make autoconf \
```
Comment (sigma-victor-bairak) [inline docker/phpfpm/DockerFile:4]: Not sure whether we need these packages within docker. I guess they are needed to run Makefile-s? All make commands are implied to be run outside container.

**Pull request merged**